test/bash/tools/kustomize.bats

```diff
@@ -0,0 +1,155 @@
+setup_file () {
```

Comment:
@Chumper can we somehow simplify the setup? looks like we've a lot duplicate stuff in that functions.

Reply:
@nikolaik please remove the bats tests